src/manifest.json

```diff
@@ -62,5 +62,8 @@
         "switchEngine": {
             "description": "__MSG_theme_generation_mode__"
         }
-    }
+    },
+    "web_accessible_resources": [
```

Comment:
Can it be avoided for MV2? I'm afraid it can trigger yet another precise review.

Reply:
Yes, of course we can make it exclusive to MV3.

Reply:
I fixed this already.